http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/VertxWebClientIT.java

```diff
@@ -199,4 +214,52 @@ private String getServiceName() {
     private WireMock wireMockClient() {
         return new WireMock(wiremock.getHost().substring("http://".length()), wiremock.getPort());
     }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    private static final class Span implements Comparable<Span> {
+
```

Comment:
Maybe we could create a variable that represents a ROOT_PARENT, just for makes it more "easy to read", and then replace all nulls by this variable.

private static final String ROOT_PARENT = null;